app/models/katello/content_view_history.rb

```diff
@@ -66,5 +66,13 @@ def humanized_status
         _("Success")
       end
     end
+
+    def description
+      if self.task.try(:label) == "Actions::Katello::ContentView::Publish"
+        content_view_version.description
```

Comment:
@daviddavis, If we don't have reason to keep them separate, combining them would be nice.  As for doing it as part of this PR or separate, I'll leave that up to you.  If the changes are small, having them together could be good for historical info.